cursorless-talon/src/actions/actions.py

```diff
@@ -43,9 +43,9 @@ def cursorless_command(action_id: str, target: dict):
         elif action_id in makeshift_action_map:
             command, command_options, talon_options = makeshift_action_map[action_id]
             return_value = (
-                vscode_command(command, target, command_options)
+                server_command(command, target, command_options)
                 if talon_options.await_command
-                else vscode_command_no_wait(command, target, command_options)
+                else server_command_no_wait(command, target, command_options)
```

Comment:
you'll get a merge conflict here; just remove it

Reply:
Sorry not clear what I need to do here?

Reply:
When you try to merge this branch with main, you'll have a merge conflict. Can just remove this code

Reply:
Will do,

Reply:
Removed